package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "codesplain-lambdas",
+  "version": "0.1.0",
```

Comment:
That is the version we are using for [codesplainUI](https://github.com/maryvilledev/codesplainUI/blob/master/package.json#L3) and we are still in beta, so it is correct as far as I'm aware.